src/hub.js

```diff
@@ -209,6 +215,7 @@ const onReady = async () => {
 
   const enterScene = async (mediaStream, enterInVR, janusRoomId) => {
     const scene = document.querySelector("a-scene");
+    scene.renderer.sortObjects = true;
     const playerRig = document.querySelector("#player-rig");
     document.querySelector("a-scene canvas").classList.remove("blurred");
```

Comment:
nit (know this wasn't changed in this commit): this can be `scene.querySelector("canvas")`